README.adoc

```diff
@@ -35,8 +35,8 @@ Learn how to create a REST service with JAX-RS, JSON-B, and Open Liberty.
 
 == What you'll learn
 
-You will learn how to build and test a simple REST service with JAX-RS and JSON-B, which will expose
-the JVM's system properties. The REST service will respond to `GET` requests made to the `\http://localhost:9080/LibertyProject/system/properties` URL.
+You will learn how to build and test a simple RESTful service with Jakarta EE RESTful Web Services (restfulWS) and JSON-B, which will expose
+the JVM's system properties. The RESTful service will respond to `GET` requests made to the `\http://localhost:9080/LibertyProject/system/properties` URL.
```

Comment:
The RESTful service will respond to `GET` requests made to the `\http://localhost:9080/LibertyProject/system/properties` URL.
--->
The RESTful service responds to `GET` requests made to the `\http://localhost:9080/LibertyProject/system/properties` URL.